Packs/Devo/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,7 @@
+#### Integrations
+##### Devo v2
+- Added fetch Alerts/Incidents pull limit option to prevent plugin from crashing.
+- Added fetch Alerts/Incidents pull from custom Devo tables.
+- Updated the Docker image to: *demisto/devo:1.0.0.61805*.
+- Fixed the issue where the plugin was not able to fetch Alerts/Incidents from Devo tables where contents in          
+  'extraData' contains None or 'null' values.
```

Comment:
Consider adding an explanation of why the fetch_incidents_deduplication parameter no longer exists